_pydev_bundle/pydev_monkey.py

```diff
@@ -1173,11 +1174,13 @@ def _get_threading_modules_to_patch():
 
 
 def patch_thread_module(thread_module):
+    attr = "_start_joinable_thread" if IS_PY313_OR_GREATER else "_start_new_thread"
+    is_start_joinable = thread_module is threading and IS_PY313_OR_GREATER
     if getattr(thread_module, "_original_start_new_thread", None) is None:
         if thread_module is threading:
-            if not hasattr(thread_module, "_start_new_thread"):
+            if not hasattr(thread_module, attr):
                 return  # Jython doesn't have it.
-            _original_start_new_thread = thread_module._original_start_new_thread = thread_module._start_new_thread
+            _original_start_new_thread = thread_module._original_start_new_thread = getattr(thread_module, attr)
```

Comment:
threading no longer has a `_start_new_thread`, but instead has a `_start_joinable_thread`. This takes two new key word arguments though, so had to handle that case below.